### ISSUE TITLE
digitsEnToFa is not defined

### ISSUE THREAD

**Issue by mbasirati** [persian-tools/persian-tools#2]
Hi Ali,
Thanks for this amazing library.  
I tested this code in html file:
```
<html>
  <head>
    <script src="https://cdn.jsdelivr.net/npm/persian-tools"> </script>
    <script type="text/javascript">    
      var convertToFa = digitsEnToFa(1234567);
    </script>
  </head>

  <body>
  </body>
</html>
```
But there is an error : 
> Uncaught ReferenceError: digitsEnToFa is not defined  

Where is the problem?

**Comment by ali-master** [comment]
Hi Dear,

Thank you for reporting, I will fix it as soon as possible.

**Comment by ali-master** [comment]
Thanks for looking into this.

It should be fixed in **v0.0.5**.
